### DISCUSSION TITLE
Modified version of logo that looks better in dark mode?

### DISCUSSION THREAD

**Discussion by ianmcook** [ibis-project/ibis#9119] [Ideas]
When the Ibis logo is set on a dark background (as on the website homepage in dark mode), the black areas lose contrast. At least to my eye, this detracts from the appearance of the logo and makes it less discernible.

I experimented with replacing the dark colors in the logo with lighter colors to see if this improved its appearance. I think it does. I'm not 100% thrilled with the result, but I think it's an improvement over the current state. What do y'all think?

The idea is that we could keep the current logo in light mode but have it switch to a modified version like this in dark mode.

### Before:
<img width="636" alt="before" src="https://github.com/ibis-project/ibis/assets/280509/f62a4fe7-e027-4c73-af61-c1fc72481f9c">

### After:
<img width="636" alt="after" src="https://github.com/ibis-project/ibis/assets/280509/859ed1f5-4830-49fc-a328-1121a05f1a4e">

**Comment by ianmcook** [comment]
Here's the source SVG if anyone else wants to experiment with this. What I did was:
- Changed the fill and stroke of the black regions from `#000000` to `#666666`, except for the eye.
- Changed the fill of the eye (the only `<ellipse />` in the file) from `#000000` to `#555555` (it looks better slightly darker).
- Changed the fill of the gray regions from `#4d4d4d` to `#bdbdbd`.
![logo-dark](https://github.com/ibis-project/ibis/assets/280509/c94e90b3-b0bc-4f49-bf3e-2e54de76f9e2)

**Reply by lostmygithubaccount** [reply]
![ibis-purp](https://github.com/ibis-project/ibis/assets/54814569/989240e1-6454-44d9-96e2-d252fb538fcd)

**Comment by ianmcook** [comment]
There's a discussion here describing how to implement separate dark mode / light mode images with Quarto: https://github.com/quarto-dev/quarto-cli/discussions/6666

**Reply by lostmygithubaccount** [reply]
I struggled trying to get this working recently (though for a slightly difference case), but it may have been a `skill issue`
